src/tilegarden/scripts/template-vars.js

```diff
@@ -6,8 +6,6 @@
 
 const fs = require('fs')
 
-const envPrefix = process.env.NODE_ENV === 'production' ? 'PROD_' : 'DEV_'
```

Comment:
Since we manage all environment variables with an `.env` file, this prefixing is no longer necessary.